src/app/core/services/github.service.ts

```diff
@@ -246,6 +246,29 @@ export class GithubService {
     octokit.issues.updateLabel({ owner: ORG_NAME, repo: REPO, name: labelName, current_name: labelName, color: labelColor });
   }
 
+  /**
+   * Checks if the given list of users are allowed to be assigned to an issue.
+   * Throws an error if at least one of them is not allowed.
+   * @param assignees - GitHub usernames to be checked
+   */
+  areUsersAssignable(assignees: string[]): Observable<void> {
+    return from(
+      octokit.issues.listAssignees({
+        owner: ORG_NAME,
+        repo: REPO
+      })
+    ).pipe(
+      map((response: { data: { login: string }[] }) => response.data.map((user) => user.login)),
+      map((list: string[]) =>
```

Comment:
```suggestion
      map((assignables: string[]) =>
```
let's use a more semantic variable name